stubs/Flask-Cors/flask_cors/decorator.pyi

```diff
@@ -0,0 +1,17 @@
+from datetime import timedelta
+from logging import Logger
+from typing import Any, Callable
+
+LOG: Logger
+
+def cross_origin(
```

Comment:
```suggestion
def cross_origin(
     *args: Any,
```
It looks like it ignores all *args. Meaningful arguments can only be passed as **kwargs.